ztrouy/cpyou
# Feature - Home View

### Purpose
To allow Users to see recent interactions other Users have had with them in the Home view

### Files Changed
- Created initial Comment DTO used for Comments in the Home view in new `CommentForHomeDTO.cs` DTO class
- Added endpoint that returns recent interactions a User has received in `CommentController.cs`
- Added fetch that gets recent interactions a User has received in `commentManager.js`
- Created file, and wrote rendering and logic for the Comment component used in the Home view in new `CommentForHome.jsx` component module
- Created file, and wrote rendering and logic for the Home view in new `Home.jsx` component module
- Added Home view component to the Home view route in `ApplicationViews.jsx`
- Added missing include in GetSingle endpoint in `BuildController.cs`
- Updated navbar to render as sticky, rather than static in `NavBar.jsx`

### To Test
Fetch, setup, and run according to project README, then confirm the following
- Home view shows up to 3 Comments that were either: a Comment left on one of your Builds and you aren't the most recent Reply, a Comment you made and the most recent Reply is not by you, a Comment you replied to and the most recent Reply is not by you
- Replying to a Comment on the Home view causes the Comment to be replaced with a new Comment that meets the previous criteria
- When there are no recent interactions on the Home view, a message appears that informs the User
- Navbar is always on the top of the screen

closes #20
